test/IRac_test.cpp

```diff
@@ -2663,6 +2727,26 @@ TEST(TestIRac, strToModel) {
   EXPECT_EQ(0, IRac::strToModel("FOOBAR", 0));
 }
 
+TEST(TestIRac, strToCommandType) {
+  EXPECT_EQ(stdAc::ac_command_t::kControlCommand,
+            IRac::strToCommandType("Control"));
+  EXPECT_EQ(stdAc::ac_command_t::kSensorTempReport,
+            IRac::strToCommandType("IFeel Report"));
+  EXPECT_EQ(stdAc::ac_command_t::kSensorTempReport,
+            IRac::strToCommandType("IFeel"));
+  EXPECT_EQ(stdAc::ac_command_t::kTimerCommand,
+            IRac::strToCommandType("Set Timer"));
+  EXPECT_EQ(stdAc::ac_command_t::kTimerCommand,
+            IRac::strToCommandType("Timer"));
+  EXPECT_EQ(stdAc::ac_command_t::kConfigCommand,
+            IRac::strToCommandType("Config"));
+  EXPECT_EQ(stdAc::ac_command_t::kControlCommand,
+            IRac::strToCommandType("FOOBAR"));
+  EXPECT_EQ(stdAc::ac_command_t::kTimerCommand,
+            IRac::strToCommandType("FOOBAR",
+            stdAc::ac_command_t::kTimerCommand));
```

Comment:
Is kTimerCommand the best 'default'? I would think 'kControlCommand' would be? As that's the common/normal/typical message type etc.

Reply:
Yes, `kControlCommand` is (by far) the best default. Since this is an UT though, and this this function's declaration is 
``` c++
strToCommandType(const char *str, ac_command_t def = ac_command_t::kControlCommand);
```

... it's the following line which tests for what you describe  (defaulting to `kControlCommand`). 
``` c++
EXPECT_EQ(stdAc::ac_command_t::kControlCommand, IRac::strToCommandType("FOOBAR"));
```
The assertion you highlighted is merely checking for a custom default value support _(which has to be different from `kControlCommand`, otherwise the test is not really checking anything)_.

As to... why does this fn have a customizable default --> this seemed to be a pattern for all the `strTo...` functions in `IRac` so I followed suit.

Reply:
Yep. That's my bad. I should have checked the underlying function first. I didn't.